addons/hr/models/hr_employee.py

```diff
@@ -187,6 +189,26 @@ def _compute_work_permit_name(self):
             permit_no = '_' + employee.permit_no if employee.permit_no else ''
             employee.work_permit_name = "%swork_permit%s" % (name, permit_no)
 
+    @api.depends('work_contact_id')
+    def _compute_contacts(self):
+        for employee in self:
+            employee.contact_ids = employee.work_contact_id
+
+    @api.depends('contact_ids')
+    def _compute_contacts_count(self):
+        for employee in self:
+            employee.contacts_count = len(employee.contact_ids)
```

Comment:
But also now.. do we really need count? won't it be 1 all the time?

Reply:
Hello @sofiagvaladze @Bertrand2 

My bad :pray:,
Since saas-16.2, I noticed that employees had two contacts showing up, mainly because of the address_home_id.
I REV the commit and didn't think about the employee's actual number of related partners. 
I found that the removal of the address_home_id field resulted in only one contact for the employee
That's why there is no need to count the partners